Containers/imaginary/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN set -ex; \
         vips-jxl \
         vips-poppler \
         build-base; \
-    go install github.com/h2non/imaginary@b632dae8cc321452c3f85bcae79c580b1ae1ed84
+    go install github.com/h2non/imaginary@b632dae8cc321452c3f85bcae79c580b1ae1ed84

```

Comment:
This looks suspicious

Reply:
Nothing suspicious in here.

The difference is the carriage return (^M) at the end of the line. Unfortunately you can not see this within Github, but you can, if you do a `git diff`

<img width="698" alt="image" src="https://github.com/nextcloud/all-in-one/assets/4036362/ef95d9ac-864e-4ad1-ac03-7b3a8d2444ff">

Where does it come from? This is what I have configured: https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings

Reply:
Can you remove the ^M again? Then I can merge

Reply:
I can, if you really want.

But first let me try again to explain, why I think it's more consistent to leave it in.

`cat -A` shows the invisible carriage return characters. Now look at the respective line before the PR. This line is the only one not including ^M (equals to '\r' or 'CR'). ^M$ means \r\n or CRLF, which can be interpreted by all platforms and therefore is preferable when multiple people collaborate.

Pre PR :
```
cat -A Containers/imaginary/Dockerfile

FROM golang:1.20.4-alpine3.17 as go^M$
RUN set -ex; \^M$
    apk add --no-cache \^M$
        vips-dev \^M$
        vips-magick \^M$
        vips-heif \^M$
        vips-jxl \^M$
        vips-poppler \^M$
        build-base; \^M$
    go install github.com/h2non/imaginary@b632dae8cc321452c3f85bcae79c580b1ae1ed84$
^M$
FROM alpine:3.17.3^M$
RUN set -ex; \^M$
    apk add --no-cache \^M$
        tzdata \^M$
        ca-certificates \^M$
        netcat-openbsd \^M$
        vips \^M$
        vips-magick \^M$
        vips-heif \^M$
        vips-jxl \^M$
        vips-poppler^M$
^M$
COPY --from=go /go/bin/imaginary /usr/local/bin/imaginary^M$
^M$
USER nobody^M$
^M$
# https://github.com/h2non/imaginary#memory-issues^M$
ENV MALLOC_ARENA_MAX=2^M$
ENTRYPOINT ["imaginary", "-p", "9000", "-return-size", "-max-allowed-resolution", "222.2"]^M$
^M$
HEALTHCHECK CMD nc -z localhost 9000 || exit 1^M$
LABEL com.centurylinklabs.watchtower.monitor-only="true"^M$
```

After PR

```
FROM golang:1.20.4-alpine3.17 as go^M$
RUN set -ex; \^M$
    apk add --no-cache \^M$
        vips-dev \^M$
        vips-magick \^M$
        vips-heif \^M$
        vips-jxl \^M$
        vips-poppler \^M$
        build-base; \^M$
    go install github.com/h2non/imaginary@b632dae8cc321452c3f85bcae79c580b1ae1ed84^M$
^M$
FROM alpine:3.17.3^M$
RUN set -ex; \^M$
    apk add --no-cache \^M$
        tzdata \^M$
        ca-certificates \^M$
        netcat-openbsd \^M$
        vips \^M$
        vips-magick \^M$
        vips-heif \^M$
        vips-jxl \^M$
        vips-poppler^M$
^M$
COPY --from=go /go/bin/imaginary /usr/local/bin/imaginary^M$
^M$
ENV PORT 9000^M$
^M$
USER nobody^M$
^M$
# https://github.com/h2non/imaginary#memory-issues^M$
ENV MALLOC_ARENA_MAX=2^M$
ENTRYPOINT ["imaginary", "-return-size", "-max-allowed-resolution", "222.2"]^M$
^M$
HEALTHCHECK CMD nc -z localhost "$PORT" || exit 1^M$
LABEL com.centurylinklabs.watchtower.monitor-only="true"^M$
```

Hopefully it's now a bit clearer. But if you want me to still use the previous version, just give me a hint.

Reply:
you have a point here. I think I fixed the problem in main and rebased the branch. Ready to go now